doc/articles/features/working-with-xaml-hot-reload.md

```diff
@@ -140,6 +140,7 @@ Hot Reload is supported by Visual Studio for WinAppSDK and provides support in u
 - The app does not update its XAML, because the port number in `RemoteControl.g.cs` is `0`.
   - Ensure you have the latest version of the Visual Studio extension installed.
   - Rebuild the app until the number is different than zero.
```

Comment:
@jeromelaban I did not see your PR before opening mine based on the feedback on Discord where it would be helpful to maybe remove the RemoteControl.g.cs details as it is not visible anymore: https://github.com/unoplatform/uno/pull/16597

Reply:
Sure, both can be merged.

Reply:
@jeromelaban https://github.com/unoplatform/uno/pull/16597 is now merged + backported.
This PR will need to be updated for the conflicts